docs/orchestrating-elastic-stack-applications/connect-to-unmanaged-resources.asciidoc

```diff
@@ -0,0 +1,84 @@
+:page_id: connect-to-unmanaged-resources
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Connect to external Elastic resources
+
+Fields like `elasticsearchRef` or `kibanaRef` are useful to automatically establish connections between applications managed by a same operator instance. It is however also possible to provide existing configuration settings and credentials in a `Secret` through the `secretName` attribute:
```

Comment:
```suggestion
Fields like `elasticsearchRef` or `kibanaRef` are useful to automatically establish connections between applications managed by the same ECK operator instance. It is however also possible to connecto to applications managed by a different ECK operator instance or to applications not managed by ECK, for example an Elastic Cloud deployment.  This can be done by providing connection details and credentials in a `Secret` through the `secretName` attribute:
```

Not sure about the exact wording but I wanted to highlight the use cases for external connections a bit more.